src/utils/constants.js

```diff
@@ -34,3 +34,17 @@ export const validationSettings = {
   errorClass: "modal__error_active",
   modalSpan: ".modal__span",
 };
+
+export const profileEditButton = document.querySelector("#profile-edit-button");
+export const profileTitleInput = document.querySelector("#person-name");
+export const profileDescriptionInput = document.querySelector(
+  "#person-description"
+);
+export const profileEditForm = document.querySelector("#profile-edit-form");
+export const cardSection = document.querySelector(".cards");
+export const cardAddButton = document.querySelector("#add-button");
+export const cardAddForm = document.querySelector("#card-add-form");
+export const editAvatarButton = document.querySelector(
+  ".profile__image-overlay"
+);
+export const editAvatarForm = document.querySelector("#avatar-edit-form");
```

Comment:
**Could be improved**

We can get any form using their **unique** attribute `name` or `id` from `document.forms`

```js

const profileForm = document.forms["profile-form"];
const cardForm = document.forms["card-form"];

```

You need to add `name` or `id` first in `index.html` to achieve that.

More info here https://developer.mozilla.org/en-US/docs/Web/API/Document/forms